app/views/storage_locations/_form.html.erb

```diff
@@ -26,6 +26,9 @@
                 <% end %>
 
                 <%= f.input :warehouse_type, label: "Warehouse Type", wrapper: :input_group, collection: StorageLocation::WAREHOUSE_TYPES %>
+
+                <%= f.input :time_zone, label: "Time Zone", wrapper: :input_group,
+                            collection: ActiveSupport::TimeZone.all.sort_by(&:name).to_h { |z| [z.name, z.tzinfo.name]} %>
```

Comment:
Note: Not necessarily advocating a specific change yet, but I think sorting all the timezones by name in whatever dropdown we use might be unwieldy for users. E.g., currently they'd see a dropdown like this:
![image](https://user-images.githubusercontent.com/934707/158030319-1506e6ff-b37b-46a4-983c-dbe105c9d67e.png)

Especially with the fact that most of the `z.name` values are cities, I was initially looking for `New York`, since that's what Linux (and macOS?) have me select when I manually select the time zone.

Possible improvements:
1) Having the identifier be UTC offset + name would make it easier to navigate east-to-west, and would co-locate most of the US+Canada zones that all(?) of our users would be looking for.
2) Instead of or in addition to the above, putting the US+Canada zones (including Hawaii + Alaska, _plus_ the special case zones (the non-NE portion of Arizona)) explicitly at the top of the list since all of our current users are in those.